config/nginx.conf.erb

```diff
@@ -16,8 +16,8 @@ http {
 	server_tokens off;
 
 	log_format l2met 'measure#nginx.service=$request_time request_id=$http_x_request_id';
-	access_log logs/nginx/access.log l2met;
-	error_log logs/nginx/error.log;
+	access_log <%= ENV['NGINX_ACCESS_LOG_PATH'] || logs/nginx/access.log %> l2met;
+	error_log <%= ENV['NGINX_ERROR_LOG_PATH'] || logs/nginx/error.log %>;
```

Comment:
did you had a chance to test this? i would expect that you need quotes around the default like this `'logs/nginx/error.log'`?

Reply:
Hey thanks for the quick reply, you are correct, I got ahead of myself. I just updated the PR so that the default log paths are in single quotes. 

Nice catch!